doc/source/extending-jupyterhub.rst

```diff
@@ -91,33 +91,39 @@ consumes a lot of memory).
     Remember `apply the changes <#applying-configuraiton-changes>`_ after changing
     your config.yaml file!
 
-Extending your software stack with s2i
---------------------------------------
+Extending your software stack with repo2docker
+----------------------------------------------
 
-s2i, also known as `Source to Image <https://github.com/openshift/source-to-image>`_,
+`repo2docker <https://github.com/jupyter/repo2docker>`_
 lets you quickly convert a GitHub repository into a Docker image that we can use
 as a base for your JupyterHub instance. Anything inside the GitHub repository
 will exist in a user’s environment when they join your JupyterHub. If you
 include a ``requirements.txt`` file in the root level of your of the repository,
-s2i will ``pip install`` each of these packages into the Docker image to be
-built. Below we’ll cover how to use s2i to generate a Docker image and how to
+repo2docker will ``pip install`` each of these packages into the Docker image to be
```

Comment:
this makes it sounds python-specific (which it mostly is right now but not in the future). We should make sure to remember to change this language when we have more fully-featured R/Julia support